lib/index.js

```diff
@@ -40,18 +21,38 @@ S3rver.prototype.resetFs = function (callback) {
   });
 }
 
+S3rver.prototype.callback = function () {
+  return new App(this.options);
```

Comment:
Haha it's called `.callback()` because I'm anticipating it eventually being a Koa wrapper. Ideally once we do Koa, we can also add a getter for `.middleware` so that the app can be mounted directly. And of course Express users can still mount the result of `.callback()`.

I think it would be good to expand the `running s3rver programmatically` section in the README that explains these things.

Reply:
Hmm... I prefer `middleware` because this library is an Express app in its current state. But I won't argue with you on any of the semantics here, **as long as you document it I will accept it**.

I can see how naming it `middleware` could be confusing to Koa users (who could be expecting a Koa middleware) while `callback` would be confusing to Express users (who would expect it to be called "middleware")  – so I think making both semantics available to users by aliasing `getMiddleware` or a `middleware` getter to this function is a good idea.

Reply:
I like the alias `getMiddleware`

Just to clarify the `middleware` getter would be the array of middleware that helpers like `koa-mount` look for when mounting subapps.